framework/src/main/java/org/checkerframework/common/wholeprograminference/WholeProgramInferenceJavaParserStorage.java

```diff
@@ -173,26 +138,17 @@ private ClassOrInterfaceAnnos getClassAnnos(
      */
     private CallableDeclarationAnnos getMethodAnnos(ExecutableElement methodElt) {
         String className = getEnclosingClassName(methodElt);
-        String file = getFileForElement(methodElt);
-        ClassOrInterfaceAnnos classAnnos =
-                getClassAnnos(className, file, ((MethodSymbol) methodElt).enclClass());
+        // Read in classes for the element.
+        getFileForElement(methodElt);
+        ClassOrInterfaceAnnos classAnnos = classToAnnos.get(className);
         CallableDeclarationAnnos methodAnnos =
                 classAnnos.callableDeclarations.get(JVMNames.getJVMMethodSignature(methodElt));
         return methodAnnos;
     }
 
-    /**
-     * Get the annotations for a formal parameter type.
-     *
-     * @param methodElt the method or constructor
-     * @param i the parameter index (0-based)
-     * @param paramATM the parameter type
-     * @param ve the parameter variable
-     * @param atypeFactory the type factory
-     * @return the annotations for a formal parameter type
-     */
     @SuppressWarnings("UnusedVariable")
```

Comment:
The "UnusedVariable" warning is not issued for overriding methods.  So you can remove this warning suppression here and elsewhere.